app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function `copy_file` does not handle exceptions that may occur during file operations. Consider adding error handling to manage cases where the target file does not exist or there are issues with file permissions.